azurerm/resource_arm_key_vault_certificate.go

```diff
@@ -349,6 +357,15 @@ func resourceArmKeyVaultCertificateRead(d *schema.ResourceData, meta interface{}
 	if contents := cert.Cer; contents != nil {
 		d.Set("certificate_data", string(*contents))
 	}
+
+	if v := cert.X509Thumbprint; v != nil {
+		x509Thumbprint, err := base64.RawURLEncoding.DecodeString(string(*v))
+		if err != nil {
+			return err
+		}
+		d.Set("thumbprint", strings.ToUpper(hex.EncodeToString(x509Thumbprint)))
```

Comment:
I'm assuming the thumbprint is case insensitive?

Reply:
I think the cluster will accept lowercase, I was trying to replicate as close as possible what the Azure CLI returns